pulsar-io/jdbc/core/pom.xml

```diff
@@ -58,6 +58,14 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>${protobuf3.version}</version>
```

Comment:
```suggestion
```

This can be a bit bikeshedding to keep consistent with other dependency patterns.

Reply:
I've pushed a commit for you - to apply suggestions and fix style violation.